packages/components/src/snackbar/stories/index.story.tsx

```diff
@@ -63,11 +69,7 @@ WithActions.args = {
 export const WithIcon: StoryFn< typeof Snackbar > = DefaultTemplate.bind( {} );
 WithIcon.args = {
 	children: 'Add an icon to make your snackbar stand out',
-	icon: (
-		<span role="img" aria-label="Icon" style={ { fontSize: 21 } }>
-			🌮
-		</span>
-	),
+	icon: <Icon style={ { fill: 'currentcolor' } } icon={ wordpress } />,
```

Comment:
Not great that this requires a `style`, but we can't really add it in the component CSS due to back-compat.

Hopefully things will be nicer once we have #40102.